www/source/partials/docs/_setting-up-a-ring.html.md.erb

```diff
@@ -4,6 +4,18 @@
 
 A “Bastion Ring” is a pattern for preventing rumor loss and a split brain in a network of Habitat Supervisors - it is highly recommended for any real environment use case. Create a minimum of 3 Supervisors and join them together. They should not run any services and they should be marked as permanent peers - their only job is to spread rumors to each other. Then, when you provision additional Supervisors pass the network address of each Supervisor running in the Bastion Ring to use as a list of “initial peers”. It’s recommended to create a Bastion Ring in any network zones which may become partitioned due to a hardware failure. For example, if you have a Habitat ring spanning multiple datacenters and clouds, each should have a bastion ring of a minimum of 3 Supervisors in addition to the Supervisors running your services.
 
+## Using a scheduler
+
+Please note, if you are using a container scheduler such as Kubernetes, Swarm, or Mesos DC/OS, or a PaaS such as CloudFoundry, you should not follow the bastion ring pattern, as the scheduler handles that level of persistence and orchestration on your behalf. 
+
+For Kubernetes, you should use the Habitat Operator for Kubernetes to ensure that the application behavior you've established for your services when you defined them with Habitat is run in a Kubernetes-native way to ensure consistent and expected behavior cross-platform. 
+
+More resources on schedulers: 
+
+- https://www.habitat.sh/get-started/kubernetes/
```

Comment:
I think if you wrap these in <> it'll work:

- <https://www.habitat.sh/get-started/kubernetes/>
- <https://www.habitat.sh/docs/best-practices/#container-orchestration>
- <https://www.habitat.sh/get-started/cloudfoundry/>